From 1f62dd91314b0ce6378aeafb2a03566002036326 Mon Sep 17 00:00:00 2001
From: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
Date: Thu, 11 Dec 2014 23:36:55 +0100
Subject: [PATCH 3/3] iprutils: Fix static build by passing the libraries in
 the right order

This mistake was causing failures like this one:

m_post.c:(.text+0x60): undefined reference to `wattr_on'

http://autobuild.buildroot.net/results/500/5004e7b230635e0605acdd17d2b7d2d01fc5075c/build-end.log

Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
[Romain: rebase on top of 2.4.5]
Signed-off-by: Romain Naour <romain.naour@openwide.fr>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 66c4942..18cea94 100644
--- a/Makefile
+++ b/Makefile
@@ -16,7 +16,7 @@ TAR = cd .. && tar -zcpf iprutils-$(UTILS_VER)-src.tgz --exclude .git --exclude
 all: iprconfig iprupdate iprdump iprinit iprdbg docs 
 
 iprconfig: iprconfig.c iprlib.o iprconfig.h
-	$(CC) $(CFLAGS) $(INCLUDEDIR) -o iprconfig iprconfig.c iprlib.o $(LDFLAGS) -lform -lpanel -lncurses -lmenu -lm
+	$(CC) $(CFLAGS) $(INCLUDEDIR) -o iprconfig iprconfig.c iprlib.o $(LDFLAGS) -lform -lpanel -lmenu -lncurses -lm
 
 iprupdate: iprupdate.c iprlib.o
 	$(CC) $(CFLAGS) $(INCLUDEDIR) -o iprupdate iprlib.o iprupdate.c $(LDFLAGS)
-- 
1.9.3

